jerry-core/ecma/base/ecma-helpers-value.c

```diff
@@ -348,11 +346,14 @@ ecma_is_value_symbol (ecma_value_t value) /**< ecma value */
 inline bool JERRY_ATTR_CONST JERRY_ATTR_ALWAYS_INLINE
 ecma_is_value_bigint (ecma_value_t value) /**< ecma value */
 {
+  #if ENABLED (JERRY_BUILTIN_BIGINT)
   return (ecma_get_value_type_field (value) == ECMA_TYPE_BIGINT);
+  #else /* !ENABLED (JERRY_BUILTIN_BIGINT) */
+  JERRY_UNUSED (value);
+  return false;
+  #endif /* ENABLED (JERRY_BUILTIN_BIGINT) */
```

Comment:
Now you can remove `ECMA_CHECK_BIGINT_IN_ASSERT`.

Also this whole can can be performed for `ecma_is_value_symbol` as well. (same for ECMA_CHECK_SYMBOL_IN_ASSERT)